init_validators.sh

```
@@ -4,23 +4,29 @@
     exit 1
 }
 
 # Set number of validators
-NUM_VALIDATORS=7
+# NUM_VALIDATORS=7
 
+NUM_VALIDATORS=5
 # Store validator IPs in array
-declare -a VALIDATOR_IPS=($1 $2 $3 $4 $5 $6 $7)
+# declare -a VALIDATOR_IPS=($1 $2 $3 $4 $5 $6 $7)
+declare -a VALIDATOR_IPS=($1 $2 $3 $4 $5)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)